pyproject.toml

```diff
@@ -42,6 +42,9 @@ grpcio =  { version = "1.33.*", optional = true }
 grpcio-tools = { version = "1.33.*", optional = true }
 watchdog = { version = "*", optional = true }
 jupyterlab = { version = "^3.0", optional = true }
+moderngl = "^5.6.3"
+moderngl-window = "^2.3.0"
+mapbox-earcut = "^0.12.10"
```

Comment:
Please bump `manimpango` to latest version `0.2.4`, because it will cause SegFaults or will not work with moderngl with wheels in place.
REF: 
- https://github.com/3b1b/manim/issues/1387
- https://github.com/3b1b/manim/issues/1391
- https://github.com/ManimCommunity/ManimPango/issues/27